specs/passenger_spec.rb

```diff
@@ -75,4 +79,48 @@
       end
     end
   end
+
+  describe "total_spend" do
+    before do
+      start_time = Time.parse('2015-05-20T12:14:00+00:00')
+      end_time = start_time + 25 * 60 # 25 minutes
+      trips = [
+        RideShare::Trip.new({cost: 10.00, rating: 3, start_time: start_time, end_time: end_time}),
+        RideShare::Trip.new({cost: 10.00, rating: 3, start_time: start_time, end_time: end_time}),
+        RideShare::Trip.new({cost: 10.00, rating: 3, start_time: start_time, end_time: end_time}),
+        RideShare::Trip.new({cost:nil, rating: 3, start_time: start_time, end_time: nil})
+      ]
+      passenger_data = {
+        id: 7,
+        name: 'Speed Passenger',
+        phone_number: '555.555.5555',
+        trips: trips
+      }
+      @passenger = RideShare::Passenger.new(passenger_data)
+    end
+    it "accurately returns the total for a specific passenger's trips" do
```

Comment:
This and the test below are testing the same thing.  I would test with no trips added, then the 1st 3, and then the last one that hasn't ended yet.